README.md

```diff
@@ -69,82 +94,155 @@ nb.scraps
 ```
 
 **scrapbook** will imply the storage format by the value type of any registered
-data translators. Alternatively, the implied storage format can be overwritten by
-setting the `storage` argument to the registered name (e.g. `"json"`) of a
-particular translator.
+data encoders. Alternatively, the implied encoding format can be overwritten by
+setting the `encoder` argument to the registered name (e.g. `"json"`) of a
+particular encoder.
 
 This data is persisted by generating a display output with a special media type
-identifying the content storage format and data. These outputs are not visible in
-notebook rendering but still exist in the document. Scrapbook then can rehydrate
-the data associated with the notebook in the future by reading these cell outputs.
+identifying the content encoding format and data. These outputs are not always
+visible in notebook rendering but still exist in the document. Scrapbook can
+then rehydrate the data associated with the notebook in the future by reading
+these cell outputs.
 
-### `sketch` to save _display output_
+#### With _display output_
 
-Display a named snap (visible display output) in a retrievable manner.
+To display a named scrap with visible display outputs, you need to indicate that
+the scrap is directly renderable.
 
-Unlike `glue`, `sketch` is intended to generate a visible display output
-for notebook interfaces to render.
+This can be done by toggling the `display` argument.
 
 ```python
-# record an image highlight
-sb.sketch("sharable_png", IPython.display.Image(filename=get_fixture_path("sharable.png")))
-# record a UI message highlight
-sb.sketch("hello", "Hello World")
+# record a UI message along with the input string
+sb.glue("hello", "Hello World", display=True)
```

Comment:
How does this deal with the case of outputs with multiple media types. 

For example, `vdom.div("hello")`  has  `text/html` (due to the `_repr_html_` method) and `text/plain` and `application/vdom.v1+json` (due to its `_repr_mimebundle_` method). Will all of them be stored and accessed via scrapbook?

Reply:
Yes. I missed this in the docs, but if display is a tuple or list it passes to IPython.core.formatters.format_display_data as the `include` argument if you wanted to filter the list of _repr_mimebundle_. (https://ipython.readthedocs.io/en/stable/api/generated/IPython.core.formatters.html)

I'll update this doc section